pennylane/devices/default_clifford.py

```diff
@@ -307,6 +336,36 @@ def circuit(state):
         >>> circuit(basis_states[2])
         tensor(0.0, requires_grad=True)
 
+    .. details::
+        :title: Error Channels
+        :href: clifford-errors
+
+        This device supports the ``finite-shot`` execution of quantum circuits with
+        the following error channels that add ``Pauli noise``, allowing for one to perform
+        any sampling-based `measurements <https://github.com/quantumlib/stim/>`_.
```

Comment:
Not sure if this is the right URL. Maybe let's just remove the hyperlink
```suggestion
        any sampling-based measurements.
```

Reply:
Ahh, wait. I had the URL for the PennyLane measurements page here.